contrib/debian/control

```diff
@@ -0,0 +1,23 @@
+Source: netdata
+Build-Depends: debhelper (>= 9),
+ dpkg-dev (>= 1.13.19),
+ dh-systemd (>= 1.5),
+ zlib1g-dev,
+ autoconf,
+ automake,
+ pkg-config
+Section: net
+Priority: optional
+Maintainer: Costa Tsaousis <costa@tsaousis.gr>
```

Comment:
@mcnewton Makes perfect sense that you want to target as many distributions as possible. That explains what I see here.

I think I will take ownership of the RFP bug on debian and start working on getting it into unstable. Your contributions will make a great base to start on.